ITE/9/README.adoc

```diff
@@ -59,6 +59,10 @@ The new attestation definition SHOULD include the following fields.
 
 This section must discuss at a high level the reason for creating the new attestation type. Essentially, it describes the problems that can be solved using the new attestation.
 
+==== Use Cases
+
+This section must discuss at least one concrete use case that will be covered by this predicate and how the current predicate types (if any) do not solve this.
```

Comment:
Is this a note to self? If so, maybe mark as `TODO` or `FIXME`?

Reply:
No this is a new section in the ITE-9

Reply:
It's a description of expected fields in a predicate proposal.

Reply:
On that note, should we combine `Purpose` and `Use Cases` into one?

Reply:
> On that note, should we combine Purpose and Use Cases into one?

Actually, I think while they're related, it's fine to have them separate at slightly different levels of detail.